recipe/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -euo pipefail
+
+cd src
+
+TCL_VERSION="$(echo 'puts [info tclversion]' | tclsh)"
+TCL_LIB_PATH="$PREFIX"/lib/tcl"$TCL_VERSION"
+BWIDGET_PATH="$TCL_LIB_PATH"/bwidget
+
+[[ ! -d "$BWIDGET_PATH" ]] || exit 1
+mkdir -p "$BWIDGET_PATH"
+cp -r * "$BWIDGET_PATH"
```

Comment:
```suggestion
mv * "$BWIDGET_PATH"/
```
Doing a move instead of a copy will save disk on the build workers.

Reply:
Fixed

Reply:
Well, we still need `src/LICENSE.txt` for the packaging step:
`ValueError: License file given in about/license_file (/var/folders/sy/f16zz6x50xz3113nwtb9bvq00000gp/T/abs_53dk_8n0cv/clone/recipe/src/LICENSE.txt) does not exist in source root dir or in recipe root dir (with meta.yaml)`

I modified commands to handle this issue